mlflow/evaluation/fluent.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+from typing import List, Optional
+
+from mlflow.entities.evaluation import Evaluation as EvaluationEntity
+from mlflow.evaluation.evaluation import Evaluation
+from mlflow.evaluation.utils import evaluations_to_dataframes
+from mlflow.tracking.client import MlflowClient
+from mlflow.tracking.fluent import _get_or_start_run
+
+
+def log_evaluations(
```

Comment:
This is a fluent API, hence we call the file `mlflow/evaluation/fluent.py`. We just aren't importing this within `mlflow/__init__.py` yet or documenting it